### PR TITLE
Meow

### PR DESCRIPTION
**Pull request by WorldMaker**
This is a rather silly PR, but I promised a few Seamonkeys I would actually do it, so here it is done: this PR replaces the boring term "Like" with the much more amusing "Meow".

### CODE REVIEW

**Comment by walkeriniraq** [comment]
Hmm - thoughts on using it as a meta-word, like smurf?  E.g. "3 seamonkeys
meow this."

**Comment by WorldMaker** [comment]
That was actually my original intent, but as you can see I chickened out at the last minute. I tried to pull it back to the "humming along with the song" thing. "3 seamonkey meow along with this" is too wordy though. "3 seamonkeys meow along"?

**Comment by Wingar** [comment]
Honestly, I was gonna do this as an easter egg. Hide somewhere, but on click it'd change all the like stuff to "Meow", like we have here.
One thing I'm not so sure about having this as default is the language. We had a problem where no one used hashtags this year, we didn't really communicate the fact they existed, I feel like this would also confuse a few people, but maybe I'm just being paranoid.

**Comment by WorldMaker** [comment]
I figure because it is a rename of an existing feature and a button on the "action" bar that it would still probably see just as much use. It would also be it's own Easter Egg given how few of us used the website directly rather than CM.

Of course, if you wanted to be extra careful you could add a useful icon like Twitter's 🌟 or Facebook-esque 👍 or cryptically 😻.

Sent from my wireless telegraph. Full stop.
________________________________
